compute_tools/src/compute.rs

```diff
@@ -256,7 +256,28 @@ impl ComputeNode {
             .unwrap_or_else(|| "5432".to_string());
         wait_for_postgres(&mut pg, &port, pgdata_path)?;
 
-        let mut client = Client::connect(&self.connstr, NoTls)?;
+        // If connection fails,
+        // it may be the old node with `zenith_admin` superuser.
+        //
+        // In this case we need to connect with old `zenith_admin`name
+        // and rename the user.
+        let mut client = match Client::connect(&self.connstr, NoTls) {
+            Err(e) => {
+                info!(
+                    "cannot connect to postgres: {}, retrying with `zenith_admin` username",
+                    e
+                );
+                let zenith_admin_connstr = self.connstr.replacen("cloud_admin", "zenith_admin", 1);
+
+                let mut client = Client::connect(&zenith_admin_connstr, NoTls)?;
+                client.simple_query("ALTER USER zenith_admin RENAME TO cloud_admin")?;
```

Comment:
@lubennikovaav, ugh, I just realized that we cannot do this rename :(

```
=# alter role alexk rename to ak;
ERROR:  session user cannot be renamed
```

You cannot rename the user used in the active session. We can try to do the following, though:
```
create role cloud_admin with login;
grant zenith_admin to cloud_admin;
```

Reply:
And also I figured out how to test it! Just for test you can use `cloud_superadmin` in the cloud repo and do the additional role create `cloud_admin` -> `cloud_superadmin` in the `compute_ctl`. That way you will test more or less the same process that will happen on prod and staging

Reply:
@ololobus, do we still need this?